paths.go

```diff
@@ -44,12 +44,11 @@ func newPaths(os, arch string, o Options) (p *Paths, err error) {
 	p.initAstilectronDirectory()
 	p.astilectronApplication = filepath.Join(p.astilectronDirectory, "main.js")
 	p.astilectronDownloadSrc = AstilectronDownloadSrc()
-	p.astilectronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("astilectron-v%s.zip", VersionAstilectron))
+	p.astilectronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("astilectron-%s-%s-v%s.zip", os, arch, VersionAstilectron))
 	p.astilectronUnzipSrc = filepath.Join(p.astilectronDownloadDst, fmt.Sprintf("astilectron-%s", VersionAstilectron))
 	p.electronDirectory = filepath.Join(p.vendorDirectory, "electron")
```

Comment:
An important part of this change is to update line 49 as well I think since you also want the place where Electron is extracted to be os/arch dependent.

Something like this may do just fine:

```go
p.electronDirectory = filepath.Join(p.vendorDirectory, "electron", os, arch)
```